modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaHttp4sClientCodegen.java

```diff
@@ -0,0 +1,459 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
i think this line can be removed per https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license

Reply:
done via https://github.com/OpenAPITools/openapi-generator/pull/19786